marklogic-data-hub/src/test/java/com/marklogic/hub/dhs/DeployUserAmpsTest.java

```diff
@@ -40,7 +40,7 @@
 
     @BeforeEach
     void checkIfTestsCanBeRun(){
-        Versions.MarkLogicVersion mlVersion = new Versions(getHubClient()).getMLVersion();
+        MarkLogicVersion mlVersion = new Versions(getHubClient()).getMarkLogicVersion();
```

Comment:
Same thing here - dump Versions in favor of new MarkLogicVersion(getHubClient().getManageClient()).